packages/replay/src/upload.ts

```diff
@@ -141,57 +140,14 @@ class ReplayClient {
     await this.client.sendCommand("Internal.reportCrash", { data });
   }
 
-  async connectionUploadRecording(recordingId: string, path: string) {
-    if (!this.client) throw new Error("Protocol client is not initialized");
-
+  async connectionUploadRecording(path: string, uploadLink: string) {
     const file = fs.createReadStream(path);
-
-    let buffer: Buffer | undefined;
-    let offset = 0;
-
-    debug("%s: Replay file size: %d bytes", recordingId, fs.statSync(path).size);
-
-    const send = async (b: Buffer) => {
-      const length = b.length;
-      debug("%s: Sending %d bytes at offset %d", recordingId, length, offset);
-
-      await new Promise<void>((resolve, reject) =>
-        this.client?.sendCommand(
-          "Internal.addRecordingData",
-          { recordingId, offset, length },
-          b,
-          undefined,
-          err => (err ? reject(err) : resolve())
-        )
-      );
-
-      offset += length;
-    };
-
-    for await (const chunk of file) {
-      const cb = chunk instanceof Buffer ? chunk : Buffer.from(chunk);
-      debug("%s: Read %d bytes from file", recordingId, cb.length);
-
-      buffer = buffer ? Buffer.concat([buffer, cb]) : cb;
-
-      if (buffer.length >= ChunkGranularity) {
-        const data = buffer.subarray(0, ChunkGranularity);
-        buffer = buffer.subarray(ChunkGranularity);
-        await send(data);
-      }
-    }
-
-    if (buffer?.length) {
-      await send(buffer);
-    }
-
-    debug("%s: Uploaded %d bytes", recordingId, offset);
-
-    // Explicitly mark the recording complete so the server knows that it has
-    // been sent all of the recording data, and can save the recording.
-    // This means if someone presses Ctrl+C, the server doesn't save a
-    // partial recording.
-    await this.client.sendCommand("Internal.finishRecording", { recordingId });
```

Comment:
We maaaye still want to call this afterward so the API knows when the CLI thinks it is done? At least then if the user disconnects without finishing uploading to S3, we'll be able to see the disconnect-without-finish and clear the DB the DB entry?

Reply:
Yeah I think I'm good with that.

Reply:
Done!